cmd/buildkitd/main.go

```diff
@@ -182,7 +182,7 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "group",
-			Usage: "group (name or gid) which will own all Unix socket listening addresses",
+			Usage: "group (name or gid) which will own all Unix socket or Windows named pipe listening addresses",
```

Comment:
Assuming this usage is not reused for docs we could show "Unix socket" or "Named pipe" based on the `runtime.GOOS`

Reply:
better yet, sure.

Reply:
Updated, let me know how that looks.